testr/testr.go

```diff
@@ -0,0 +1,116 @@
+/*
+Copyright 2019 The logr Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Package testr provides support for using logr in tests.
+package testr
+
+import (
+	"testing"
+
+	"github.com/go-logr/logr"
+	"github.com/go-logr/logr/funcr"
+)
+
+// NewTestr returns a logr.Logger that prints through a testing.T object.
+// Info logs are only enabled at V(0).
+func NewTestr(t *testing.T) logr.Logger {
```

Comment:
I'm wondering: can or should this be `testr.New` instead of `testr.NewTestr`? The latter makes sense if there are other things that can be created in a package, but this isn't the case here. Or is it always better to be more explicit?

Reply:
Good point, funcr uses `New`.  Will fix